### PR TITLE
fix(message): fix message enter & leave flash issue

### PR DESCRIPTION
**Pull request by oceanlvr**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/master/.github/CONTRIBUTING.md)

#### What is the current behavior?

Flash animation because of `vue-virtual-scroller`

#### Issue Number

None.

#### What is the new behavior?

Remove enter and leave animation temporally. Keep the list of highest performance first.(and better compatibility)

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Specific Instructions

Are there any specific instructions or things that should be known before review?

### CODE REVIEW

**Comment by oceanlvr** [comment]
WIP

- [ ] retain message flash / mis-render in messageList

File: src/views/connections/ConnectionsDetail.vue [hunk 1]
```diff
@@ -1053,6 +1053,9 @@ export default class ConnectionsDetail extends Vue {
     return async (topic: string, payload: Buffer, packet: SubscriptionModel) => {
       const convertPayload = this.convertPayloadByType(payload, this.receivedMsgType, 'receive') as string
       const receviedPayload = this.convertPayloadByScript(convertPayload, 'publish')
+      console.log('======payload===============')
+      console.log(receviedPayload)
+      console.log('====================================')
```
Comment (ysfscream) [inline src/views/connections/ConnectionsDetail.vue:1058]: These are debugging information, no need to merge them into master?

File: src/views/connections/ConnectionsDetail.vue [hunk 2]
```diff
@@ -1101,6 +1104,11 @@ export default class ConnectionsDetail extends Vue {
         if (isFromActiveTopic || isFromNotActiveTopic) {
           this.$log.info(`Message Arrived with topic: ${topic}`)
           this.messages.push(receivedMessage)
+          console.log('===hi!====')
+          console.log('====================================')
+          console.log(this.record.messages)
+          console.log(this.messages)
+          console.log('====================================')
```
Comment (ysfscream) [inline src/views/connections/ConnectionsDetail.vue:1111]: 👆same

Comment (oceanlvr) [reply src/views/connections/ConnectionsDetail.vue:1111]: sorry for that, re-commit this.

**Review by ysfscream — APPROVED** [review]

**Pull request merged**